rust_src/src/fonts.rs

```diff
@@ -0,0 +1,71 @@
+use remacs_macros::lisp_fn;
+use remacs_sys::{font, Qfont_spec, Qfont_entity, Qfont_object, wrong_type_argument};
+use lisp::LispObject;
+use symbols::intern;
+use vectors::LispVectorlikeRef;
+
+// A font is not a type in and of itself, it's just a group of three kinds of
+// pseudovector. This newtype allows us to define methods that yield the actual
```

Comment:
This could be improved. 
* plural of pseudovectors 
* missing space between new and type.
* First you mention that a font is not a type and then you continue with "This new type"